### ISSUE TITLE
Pasting from the clipboard missing and replaced characters (Italian language)

### ISSUE THREAD

**Issue by MX9000** [joncampbell123/dosbox-x#4884]
### Describe the bug

I have a notebook with Italian keyboard and Italian Windows 10.
The output of DOSBox-X KEYB command is "Codepage 437 has been loaded for layout it".
I also tried changing the codepage from 437 to 850 because I found that Italian MS-DOS 6 by default installs that, but nothing changed.
When I paste a test string from Notepad to DOSBox-X prompt some characters are ignored or replaced: see the attached txt file for details.
[Clipboard test string and issues.txt](https://github.com/joncampbell123/dosbox-x/files/14544294/Clipboard.test.string.and.issues.txt)

### Steps to reproduce the behaviour

1. Paste the test string in DOSBox-X prompt
2. See the missing and replaced characters

### Expected behavior

All characters supported by active codepage should be copied

### What operating system(s) this bug have occurred on?

Windows 10 Pro 22H2 Italian

### What version(s) of DOSBox-X have this bug?

2024.03.01, Visual Studio SDL1 & SDL2 32-bit

### Additional information

Probably related bugs:
- wrong paste chars #4336
- Clipboard copy of characters missing in DOS #4344

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by rderooy** [comment]
Did you check if the characters exist in the DOS codepage? Especially CP437 is unlikely to have the characters, and even 850 may not have all characters.

You can check the characters that are in the codepages here:
- https://en.wikipedia.org/wiki/Code_page_437
- https://en.wikipedia.org/wiki/Code_page_850

**Comment by MX9000** [comment]
@rderooy yes, I see that codepages 437 and 850 have at least the characters "-" "." and "\".
I discovered this issue by accident trying to copy commands from the Wiki: https://dosbox-x.com/wiki/Guide%3AInstalling-FreeDOS

**Comment by maron2000** [comment]
`clip_paste_bios=true` option works much better when pasting Japanese text.
Maybe also for other non-us codepages.
You can see that even alphabets have flaws, when Japanese characters are mixed.
 (e.g. PC-98 -> 1C-98 in the first sentence)

`clip_paste_bios=default`
![default](https://github.com/joncampbell123/dosbox-x/assets/68574602/ae1f12bf-38b1-40c3-b63e-304baec8b64b)

`clip_paste_bios=true`
![true](https://github.com/joncampbell123/dosbox-x/assets/68574602/72cc1700-e0eb-4921-8eaa-f694b09bded8)

Original text
Neko Project 21/W（np21w, ねこープロジェクト21/W）はPC-98エミュレータ Neko Project IIおよびNeko Project 21（ねこープロジェクトII、ねこープロジェクト21）をベースに、ニッチなハードウェアのエミュレーションやPC-98後期モデル（1993～2000頃）の再現に耐えうる改造を施したものです

**Comment by maron2000** [comment]
`clip_paste_bios=true` option works better for the text provided.
![it](https://github.com/joncampbell123/dosbox-x/assets/68574602/05b3ae26-b295-4ff8-a04d-9356d3cfac1a)
